README.md

```diff
@@ -249,6 +252,9 @@ sequenceDiagram
    this missing depenedency, and warn accordingly.
  * whilst it would be useful for end users, library requirements will not currently be factored into the `composer.json`
    dependencies, due to the complexity of different package managers on different platforms.
+ * The package will be checked to see if it is compatible with ZTS mode. If using a `--with-php-config` with ZTS
+   enabled, but the extension specifies `"support-zts": false` in its `composer.json` manifest, the installation will
+   be halted with an error explanation.
```

Comment:
nit: these lines are also too long :-)

Reply:
I will reflow the whole doc after #9 gets merged, to avoid having to rebase :)